Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            bool calculateAgain = true;
+
+            while (calculateAgain)
+            {
+                Console.WriteLine("BMI Calculator");
+
+                string name = GetString("Enter your name: ");
+                int age = GetInt("Enter your age: ");
+                float weight = GetFloat("Enter your weight(kg): ");
+                float height = GetFloat("Enter your height(cm): ");
+
+                Console.WriteLine(name + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+                float BMI = CalculateBMI(weight, height / 100);
```

Comment:
Convention is to use bmi and CalculateBmi here.
You may ask yourself why CalculateBmi:
CalculateBMIFromPerson would be troublesome then.